htslib/vcfutils.h

```diff
@@ -26,6 +26,7 @@ DEALINGS IN THE SOFTWARE.  */
 #define HTSLIB_VCFUTILS_H
 
 #include "vcf.h"
+#include "htslib/kbitset.h"
```

Comment:
Changing includes ⇒ changing dependencies in the makefile.  Also what I was suggesting in _Example of declaring…_ in kbitset.h was to not include the kbitset header here, but for this header to have

    struct kbitset;

here (actually, just below the `extern "C"` bit) and

    void bcf_remove_alleles_set(const bcf_hdr_t *header, bcf1_t *line, const struct kbitset_t *rm_set);
                                                                             ^^^^^^^^^^^^^^^^

below.  Only _vcfutils.c_ actually needs to `#include <htslib/kbitset.h>` and correspondingly have `$(htslib_kbitset_h)` in its objects' dependencies.